### PR TITLE
[bitnami/supabase] Support for customizing standard labels

### PR DESCRIPTION
**Pull request by juan131**
### Description of the change

This PR follows up #18154 ensuring this chart is adapted to use new helpers' format, hence adding support for customizing standard labels.

### Benefits

User can customize standard labels.

### Possible drawbacks

None

### Applicable issues

N/A

### Additional information

#18154 must be merged **before** this PR

### Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/supabase/templates/studio/deployment.yaml [hunk 1]
```diff
@@ -37,14 +34,8 @@ spec:
         checksum/extra-configmap: {{ include (print $.Template.BasePath "/studio/extra-configmap.yaml") . | sha256sum }}
         {{- end }}
         checksum/jwt-secret: {{ include (print $.Template.BasePath "/jwt-secret.yaml") . | sha256sum }}
-      labels: {{- include "common.labels.standard" . | nindent 8 }}
+      labels: {{- include "common.labels.standard" ( dict "customLabels" $podLabels "context" $ ) | nindent 8 }}
```
[the remark this answers is not in the document]
Comment (juan131) [reply bitnami/supabase/templates/studio/deployment.yaml:38]: fixed at https://github.com/bitnami/charts/pull/18751/commits/1058280d2f4f8f94b90ba32085394e4f60708f4c

**Review by fmulero — CHANGES_REQUESTED** [review]
Could you update the dependencies?

**Comment by juan131** [comment]
Let's wait for Kong to be released with the required changes

**Comment by juan131** [comment]
Kong released! Unblocking PR

**Review by fmulero — APPROVED** [review]
LGTM

**Pull request merged**